packages/Sortable/src/components/Item/Item.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+import HandleIcon from "@paprika/icon/lib/DragHandle";
+import { Draggable } from "react-beautiful-dnd";
+import Button from "@paprika/button";
+import SortableContext from "../../SortableContext";
+import { itemStyles, itemIndexStyles, itemHandleStyles, itemBodyStyles, itemCloseStyles } from "./Item.styles";
+
+const propTypes = {
+  child: PropTypes.node.isRequired,
+  index: PropTypes.number.isRequired,
+};
+
+const Item = ({ child, index }) => {
+  const { hasIndexes, onRemove } = React.useContext(SortableContext);
```

Comment:
Just curious: Why do we need context? Couldn't `Sortable` pass these as prop to `Item`?

Reply:
^ agree maybe drilling props might be a better solution for this component.

Reply:
+1. Could make things easier

Reply:
That's a lot of consensus.  Perhaps this hammer was too big for this nail.